src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/settings.html.twig

```diff
@@ -0,0 +1,166 @@
+<style>
+#open-settings {
+    color: var(--text-muted);
+    display: block;
+    margin: 15px 15px 5px;
+}
+
+.modal-wrap {
+    -webkit-transition: all 0.3s ease-in-out;
+    align-items: center;
+    background: rgba(0, 0, 0, 0.8);
+    display: flex;
+    height: 100%;
+    justify-content: center;
+    left: 0;
+    opacity: 1;
+    overflow: auto;
+    position: fixed;
+    top: 0;
+    transition: all 0.3s ease-in-out;
+    visibility: hidden;
+    width: 100%;
+    z-index: 100000;
+}
+.modal-wrap.visible {
+    opacity: 1;
+    visibility: visible;
+}
+.modal-wrap .modal-container {
+    box-shadow: 0 0 0 2px var(--page-background);
+    color: var(--base-6);
+    margin: 1em;
+    max-width: 94%;
+    width: 600px;
+}
+
+.modal-wrap .modal-header {
+    background: var(--table-header);
+    padding: 15px 30px;
+}
+.modal-wrap .modal-header h3 {
+    color: var(--base-6);
+    margin: 0;
+}
+
+.modal-wrap .modal-header .close-modal {
+    background: transparent;
+    border: 0;
+    cursor: pointer;
+}
+
+.modal-wrap .modal-header .close-modal:hover { opacity: 1; }
+
+.modal-wrap .modal-content {
+  background: var(--base-1);
+  margin: 0;
+  padding: 15px 30px;
+  width: 100%;
+  z-index: 100000;
+}
+.modal-content h4 {
+    border-bottom: var(--border);
+    margin: 0 0 15px;
+    padding: 0 0 5px;
+}
+.modal-content input, .modal-content .form-help {
+    margin-left: 15px;
+}
+.modal-content label {
+    cursor: pointer;
+    font-size: 16px;
+    margin-left: 3px;
+}
+.modal-content .form-help {
+    color: var(--color-muted);
+    font-size: 14px;
+    margin: 5px 0 15px 33px;
+}
+.modal-content .form-help + h4 {
+    margin-top: 45px;
+}
+
+.modal-footer {
+    background: var(--base-0);
+    padding: 15px 30px;
+    text-align: right;
+    z-index: 100000;
+}
+</style>
+
+<a href="#" id="open-settings" data-triger="profiler-settings">Settings</a>
```

Comment:
semantically, this should be a `<button type="button">`, not a link (not sure our profiler UI is accessible yet though)